mobile/src/packages/journal/journal-api.ts

```diff
@@ -28,6 +33,40 @@ class JournalApi extends BaseHttpApi {
 
     return await response.json<JournalEntryGetAllResponseDto>();
   }
+
+  public async createJournalEntry(
```

Comment:
```suggestion
  public async create(
```
Check A5 rule

Reply:
@fshabanov we have the same names for these functions in already merged frontend part

Reply:
we need to change this because of quality criteria, we will this on frontend too